### DISCUSSION TITLE
Where/what is the utils module used in the tutorials?

### DISCUSSION THREAD

**Discussion by mattroos** [arogozhnikov/einops#132] [Q&A] [answered]
In the tutorials, several things are imported from a ```utils``` module. E.g., in tutorial 1:
```python
# Examples are given for numpy. This code also setups ipython/jupyter
# so that numpy arrays in the output are displayed as images
import numpy
from utils import display_np_arrays_as_images
display_np_arrays_as_images()
```
Where/what is the ```utils``` modules?

**Comment by arogozhnikov** [comment, marked as answer]
Hi @mattroos 

It is in the same folder as tutorial itself:

https://github.com/arogozhnikov/einops/blob/master/docs/utils/__init__.py
